onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -3572,17 +3572,175 @@ def aten_max_pool1d_with_indices(
     raise NotImplementedError()
 
 
+@torch_op("aten::max_pool2d", trace_only=True)
 def aten_max_pool2d(
-    self: TensorType,
+    self: Union[DOUBLE, FLOAT, FLOAT16, INT8, UINT8],
```

Comment:
I would create a TypeVar for this (like TFloat, etc ... that we have) so we know the input type and the output type are the same

Reply:
TFloatOrUInt8 is created, 'UInt8' means Uint8 and Int8.